tutorials/source-modeling/plot_dipole_orientations.py

```diff
@@ -162,12 +156,9 @@
                             loose=1.0)
 
 # Show the three dipoles defined at each location in the source space
-dip_dir = inv['source_nn'].reshape(-1, 3, 3)
```

Comment:
@GuillaumeFavelier you don't use the inv anymore in the tutorial

Reply:
It's used at line [171](https://github.com/mne-tools/mne-python/blob/157eabc4c3784be4bc468a688ced38d94e39877f/tutorials/source-modeling/plot_dipole_orientations.py#L171), that's why I couldn't remove it. I will refactor so that it's more visible.